cmd/spanner-autoscaler/main.go

```diff
@@ -46,29 +47,28 @@ var (
 
 func init() {
 	//nolint:errcheck
-	clientgoscheme.AddToScheme(scheme)
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	//nolint:errcheck
-	spannerv1alpha1.AddToScheme(scheme)
+	utilruntime.Must(spannerv1alpha1.AddToScheme(scheme))
 
 	// +kubebuilder:scaffold:scheme
 }
 
 var (
-	metricsAddr       = flag.String("metrics-addr", "localhost:8090", "The address the metric endpoint binds to.")
-	healthzAddr       = flag.String("healthz-addr", "localhost:8091", "The address the healthz endpoint binds to.")
-	scaleDownInterval = flag.Duration("scale-down-interval", 55*time.Minute, "The scale down interval.")
+	metricsAddr          = flag.String("metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	probeAddr            = flag.String("health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
```

Comment:
If you don't have a strong motivation, it might be better to use the same flags and ports.

Reply:
These were the default values generated by `kubebuilder` v3. These port numbers are also configured in the k8s manifests automatically, so instead of changing them here and then everywhere else, I just kept the default ones. Same goes for the flags as well. 
We can revert to  the old ones, but I think it is better to move closer to the default `kubebuilder` configuration for easier maintenance and migration in future.

Ref: https://book.kubebuilder.io/migration/manually_migration_guide_v2_v3.html#rename-the-manager-flags

Reply:
I understood, thanks!
Also, this change might not be a problem because people will probably use the manifest file generated by kustomize.